runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/stubs/IsolationChannel.java

```diff
@@ -44,7 +44,7 @@
  * that each active rpc has its own channel.
  */
 @Internal
-class IsolationChannel extends ManagedChannel {
+public class IsolationChannel extends ManagedChannel {
```

Comment:
does this still need to be public?

Reply:
yea its used in GrpcWindmillServer and will be used in StreamingEngineClient when we eventually inject it.

We could create a Factory class to create the ChannelCache?